payroll-processor-functions/src/payroll-processor-functions/Features/Employees/Employee.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PayrollProcessor.Functions.Features.Employees
+{
+    public class Employee
+    {
+        public Guid Id { get; set; }
+        public string Department { get; set; } = "";
+        public DateTimeOffset EmploymentStartedOn { get; set; }
+        public string FirstName { get; set; } = "";
+        public string LastName { get; set; } = "";
+        public string Phone { get; set; } = "";
+        public string Status { get; set; } = "";
+        public string Title { get; set; } = "";
+        public IEnumerable<EmployeePayroll> Payrolls { get; set; } = new EmployeePayroll[] { };
```

Comment:
Nullable reference type protection

Reply:
Always good to initialize objects, noting you could use `Enumerable.Empty<EmployeePayrolls>();`